lib/skate/settings/user.ex

```diff
@@ -37,7 +37,7 @@ defmodule Skate.Settings.User do
   @doc """
   Update the user with the given email if one exists, otherwise insert a new one.
```

Comment:
suggestion(non-blocking): Hmm, I'm wondering if we should update this comment?
```suggestion
  Update or create the user with the given email.
```

Reply:
Good point, that's a more accurate explanation now.